tools/xcms_macro/lib.r

```diff
@@ -151,12 +151,14 @@ getPlotChromPeakDensity <- function(xdata, param = NULL, mzdigit=4) {
 
     group_colors <- brewer.pal(length(unique(xdata$sample_group)), "Set1")
     names(group_colors) <- unique(xdata$sample_group)
+	col_per_samp <- as.character(xdata$sample_group)
+	for(i in 1:length(group_colors)){col_per_samp[col_per_samp==(names(group_colors)[i])]<-group_colors[i]}
 
```

Comment:
There is an indentation issue here.
Can you use 4 spaces instead of tabulation?